### DISCUSSION TITLE
Will customtkinter add a scrollbar?

### DISCUSSION THREAD

**Discussion by JardarIversen** [TomSchimansky/CustomTkinter#904] [Ideas]
I know it's possible using tkinter, but I would like to stick to customtkinter only for the scaling, color themes, etc. There is one in the textbox but as far as i know i can't put widgets into it. **So are there plans to implement scrollbars for frames in the future?**

**Comment by TomSchimansky** [comment]
Yes I will do this, but I cannot say when, maybe in January.
